src/main/java/com/epam/jpop/bookms/BookMsApplication.java

```diff
@@ -0,0 +1,26 @@
+package com.epam.jpop.bookms;
+
+import io.swagger.v3.oas.models.OpenAPI;
+import io.swagger.v3.oas.models.info.Info;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+
+@SpringBootApplication
+@EnableWebMvc
```

Comment:
Dont need this

Reply:
removed unnecessary annotation @EnableWebMvc